src/api/tests/service/shifu/test_admin_course_detail.py

```
@@ -968,6 +1073,205 @@ def test_get_operator_course_chapter_detail_rejects_blank_params_with_params_err
+            user_bid="student-1",
+            paid_price="99.00",
+            created_at=datetime(2026, 4, 5, 8, 0, 0),
+        )
+        _seed_course_permission(
+            shifu_bid="course-detail",
+            user_bid="student-2",
+            created_at=datetime(2026, 4, 2, 8, 30, 0),
+        )
+        _seed_user_token(
+            user_bid="student-1",
+            token="token-1",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "token"

(S106)